.gitignore

```diff
@@ -163,3 +162,4 @@ pip-log.txt
 
 # Mac crap
 .DS_Store
+.idea
```

Comment:
What is a .idea? Why should we be ignoring it if it's not a generated filetype in the repo?